assets/js/views/_paymentCategories.js

```diff
@@ -0,0 +1,63 @@
+OpenDisclosure.PaymentCategoryView = Backbone.View.extend({
```

Comment:
It would be better to name this `OpenDisclosure.Views.PaymentCategory` to be in parallel with the other view names. (It's on my todo list to also namespace our Collections and Models)